.travis.yml

```diff
@@ -0,0 +1,36 @@
+language : python
+cache: pip
+
+sudo: true
```

Comment:
Does it work without specifying `sudo: true`? 

https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration
> Soon we will run all projects on the virtual-machine-based infrastructure, the sudo keyword will be fully deprecated.